test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
[OPTIONAL] Well done on adding this file to gitignore  but since ignored it should not appear in github you can use this command to remove it from git cache $ git rm –cached <filename>” command. and always add files you want to ignore to gitignore before you add them to your code base